borg/testsuite/archiver.py

```diff
@@ -1288,13 +1317,13 @@ def _test_recreate_interrupt(self, change_args, interrupt_early):
         if interrupt_early:
             process_files = 0
         with patch.object(ArchiveRecreater, 'process_item', self._recreate_interrupt_patch(process_files)):
-            self.cmd('recreate', '-sv', '--list', self.repository_location, 'input/dir2')
+            self.cmd('recreate', '-s', '--list', self.repository_location, 'input/dir2')
```

Comment:
this one is ok (output isn't used). In fact, you can remove '-s', '--list' entirely.